front/app/modules/commercial/analytics/admin/containers/Visitors/index.tsx

```diff
@@ -49,8 +109,24 @@ const Visitors = () => {
           onChangeTimeRange={handleChangeTimeRange}
           onProjectFilter={handleProjectFilter}
           onChangeResolution={setResolution}
+          showAllTime={!hasPartialVisitorData}
         />
       </Box>
+      {hasPartialVisitorData && (
+        <Box p="10px">
+          <Warning
+            text={
+              <Text color="primary" m="0px" fontSize="s">
+                {formatMessage(messages.dateInfo, {
+                  date: moment(
+                    countData.first_dimension_date_first_action_date
+                  ).format('MM/DD/YYYY'),
```

Comment:
Will this adjust for different languages? In Belgium for exampl, MM/DD/YYYY is very uncommon. DD/MM/YYYY is much more common. So that could be confusing.